### PR TITLE
Use the `until` framework for resolving results

### PR DESCRIPTION
**Pull request by th0114nd**

### CODE REVIEW

File: lib/rest_5.js [hunk 1]
```diff
@@ -484,20 +484,14 @@ function* resolveResult(result, chainId, node) {
...
-        res = yield getBlocResults(res.map(r => {return r.hash}), chainId, false, node);
-	yield promiseTimeout(1000);
-	count++;
-    }
-
-    if(count >= 60) {
-        throw new HttpError400('Transaction did not resolve');
+function* resolveResults(results) {
+    const pred = res => !res.some(r => r.status === constants.PENDING)
+    const act = node => getBlocResults(results.map(r => r.hash), false, node)
```
Comment (dustinnorwood) [inline lib/rest_5.js]: Missing chainId parameter

Comment (th0114nd) [reply lib/rest_5.js]: +1

Comment (dustinnorwood) [reply lib/rest_5.js]: Also realizing the arguments were out of order in the old version too

Comment (th0114nd) [reply lib/rest_5.js]: https://github.com/blockapps/blockapps-rest/pull/132/files

**Pull request merged**